### DISCUSSION TITLE
Nur zum Ernten zu verwenden

### DISCUSSION THREAD

**Discussion by Maag000** [Mogli12/AIVehicleExtension#276] [General]
Hallo Mogli,

Erst einmal danke für den Mod!
Leider funktioniert er nur zum ernten, nicht aber beim Säen, grubbern, düngen. 
Meldung gleich nach dem Start des Helfers:
"Der Helfer hat die Arbeit unplanmässig beendet - Grund unbekannt"

Was mache ich falsch?

**Comment by Mogli12** [comment]
Welches Gerät? Knicklenkung? Welche Karte? Hof Bergmann?

**Comment by Maag000** [comment]
Karte ist Przemasowo.
Es betrifft alle an Traktoren angehängte Maschinen. 
Bsp.: Claas Xerion 5000-4000 mit Bredal k165 
oder mit Agro Matz Rewe 8200 oder mit Amazone Condor 15001.
Diese nur als Beispiel.

**Comment by Mogli12** [comment]
Und auf Felsbrunn tritt das auch auf?

**Comment by Maag000** [comment]
Ja leider

**Comment by Mogli12** [comment]
Da liegt sicher ein Mod-Konflikt vor. Ich teste den Mod regelmäßig auf Felsbrunn.

**Comment by Maag000** [comment]
So, gerade auf Felsbrunn selbst versucht mit folgenden Equipment:

Fendt 1050 mit 517PS
Bredal K165 Düngestreuer erweiterte Breite auf 18m
Nur AI Vehicle Extension Mod 0.0.7.0 aktiviert
Feld 13

Und es funktioniert nicht.
Es wird auch Arbeitsbreite 0.00m angezeigt.

Edit:
Gerade mit Düngestreuer Kuhn Axis 40.2 M-EMC-W versucht. Hier fährt der zumindest mal los. Aber dann total unkontrolliert. Auch auf angrenzende Felder. So leider nicht brauchbar. Schade.

**Comment by Mogli12** [comment]
Beim Bredal hängt die Streubreite vom Inhalt ab. Das wird leider nicht unterstützt. 

Nimm doch einfach courseplay.

**Comment by Mogli12** [comment]
Probier mal Version 0.0.7.1. Ob allerdings der Bredal unterstützt wird, kann ich nicht sagen
